systemtest/src/test/java/io/strimzi/systemtest/kafka/listeners/MultipleListenersST.java

```diff
@@ -0,0 +1,353 @@
+
+            LOGGER.info("Generating {} listener", kafkaListenerType.name());
+
+            List<GenericKafkaListener> testCaseListeners = new ArrayList<>(5);
+
+            switch (kafkaListenerType) {
+                case NODEPORT:
+                    stochasticCount = ThreadLocalRandom.current().nextInt(2, 5);
+
+                    for (int j = 0; j < stochasticCount; j++) {
+
+                        boolean stochasticCommunication = ThreadLocalRandom.current().nextInt(2) == 0;
+
+                        testCaseListeners.add(new GenericKafkaListenerBuilder()
+                            .withName(generateRandomListenerName())
+                            .withPort(10900 + j)
+                            .withType(KafkaListenerType.NODEPORT)
+                            .withTls(stochasticCommunication)
+                            .build());
+                    }
+                    break;
+                case LOADBALANCER:
+                    stochasticCount = ThreadLocalRandom.current().nextInt(2, 3);
+
+                    for (int j = 0; j < stochasticCount; j++) {
+
+                        boolean stochasticCommunication = ThreadLocalRandom.current().nextInt(2) == 0;
+
+                        testCaseListeners.add(new GenericKafkaListenerBuilder()
+                            .withName(generateRandomListenerName())
+                            .withPort(11900 + j)
+                            .withType(KafkaListenerType.LOADBALANCER)
+                            .withTls(stochasticCommunication)
+                            .build());
+                    }
+                    break;
+                case ROUTE:
+                    testCaseListeners.add(new GenericKafkaListenerBuilder()
+                        .withName(generateRandomListenerName())
+                        .withPort(12091)
+                        .withType(KafkaListenerType.ROUTE)
+                        // Route or Ingress type listener and requires enabled TLS encryption
+                        .withTls(true)
+                        .build());
+                    break;
+                case INTERNAL:
+                    stochasticCount = ThreadLocalRandom.current().nextInt(2, 4);
+
+                    for (int j = 0; j < stochasticCount; j++) {
+
+                        boolean stochasticCommunication = ThreadLocalRandom.current().nextInt(2) == 0;
+
+                        testCaseListeners.add(new GenericKafkaListenerBuilder()
+                            .withName(generateRandomListenerName())
+                            .withPort(13900 + j)
+                            .withType(KafkaListenerType.INTERNAL)
+                            .withTls(stochasticCommunication)
+                            .build());
+                    }
+            }
+            testCases.put(kafkaListenerType, testCaseListeners);
+        }
+
+        LOGGER.info("Finished will generation of test cases for multiple listeners");
```

Comment:
```suggestion
        LOGGER.info("Finished with generation of test cases for multiple listeners");
```
and maybe log the listeners generated, if it's not logged elsewhere?